### PR TITLE
Added Komikcast to Repo

### PR DESCRIPTION
**Pull request by fenismonjaa**
Komikcast.com well-known manga reading site in Indonesia, 
already beautified using prettier.
for features (not all included in pictures) : 
Features

### CODE REVIEW

**Comment by fenismonjaa** [comment]
fell free to ask,

**Review by EncryptedDev — COMMENTED** [review]
Resolve ESlint issues

File: websites/K/Komikcast/presence.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  else if (document.location.pathname.endsWith("/bookmark/"))
+    presenceData.details = "Viewing Bookmark";
+  else if (document.location.pathname.endsWith("/download-aplikasi-komikcast/"))
+    presenceData.details = "Download APK Komikcast";
+  // Added button
+  else if (document.location.pathname.startsWith("/komik/")) {
+    const name = document
+        .querySelector(".komik_info-content-body-title")
+        .textContent.replace(/\t|\n/g, ""),
+      type = document
```
Comment (EncryptedDev) [inline websites/K/Komikcast/presence.ts:31]:
```suggestion
    presenceData.state = document
        .querySelector(".komik_info-content-body-title")
        .textContent.replace(/\t|\n/g, "");
      const type = document
```

**Comment by fenismonjaa** [comment]
presence not working after code changes, any solution?

**Comment by fenismonjaa** [comment]
> Resolve ESlint issues

what do you mean?

**Comment by Slowlife01** [comment]
Revert the last 2 commits..

**Comment by fenismonjaa** [comment]
> Revert the last 2 commits..

done

**Review by Slowlife01 — APPROVED** [review]
lgtm

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**